### PR TITLE
Fix/APP-1868 - Voting option after revote

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- Fixes issue where the voting options are still shown on voting terminal after the user has replaced their vote.

Task: [APP-1868](https://aragonassociation.atlassian.net/browse/APP-1868)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1868]: https://aragonassociation.atlassian.net/browse/APP-1868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/context/proposalTransaction.tsx [hunk 1]
```diff
@@ -129,10 +129,14 @@ const ProposalTransactionProvider: React.FC<Props> = ({children}) => {
   );
 
   const shouldDisableCallback = useMemo(() => {
-    if (voteProcessState === TransactionState.SUCCESS) return false;
+    if (
```
Comment (WissenIstNacht) [inline packages/web-app/src/context/proposalTransaction.tsx:132]: Btw, this memo does pretty much the same amount of work to check if the dependencies have change then to execute the code :)

File: packages/web-app/src/hooks/useWalletCanVote.tsx [hunk 2]
```diff
@@ -87,5 +87,9 @@ export const useWalletCanVote = (
     proposalId,
   ]);
 
-  return {data, error, isLoading};
+  return {
+    data: Array.isArray(data) ? data.some(v => v) : data,
```
Comment (WissenIstNacht) [inline packages/web-app/src/hooks/useWalletCanVote.tsx:91]: Makes sense 👍🏼

File: packages/web-app/src/pages/proposal.tsx [hunk 3]
```diff
@@ -78,7 +78,7 @@ import {Action, ProposalId} from 'utils/types';
 // TODO: @Sepehr Please assign proper tags on action decoding
 // const PROPOSAL_TAGS = ['Finance', 'Withdraw'];
 
-const PENDING_PROPOSAL_STATUS_INTERVAL = 1000 * 60 * 60;
+const PENDING_PROPOSAL_STATUS_INTERVAL = 1000 * 60;
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/proposal.tsx:81]: This seems to change back and forth with every other PR 🤔

**Review by WissenIstNacht — APPROVED** [review]
Cheers!

File: packages/web-app/src/pages/proposal.tsx (hunk 3 above)
Comment (Fabricevladimir) [reply packages/web-app/src/pages/proposal.tsx:81]: Yeah it ain't working.

File: packages/web-app/src/context/proposalTransaction.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/context/proposalTransaction.tsx:132]: I don't know what you're talking about. Ostrich algorithm for the win. Seriously though, quite a few components do need to be reviewed for that kind of thing.

**Pull request merged**